FIRBoundaries.dat

```diff
@@ -30998,4 +31024,4 @@ YYWE|0|0|42|-60.0|75.0|-2.0|163.0|-37.810595|143.53974
 -38.000491|144.37864
 -38.009167|144.389722
 -38.143333|144.363611
--38.770556|143.263889

```

Comment:
This line should be reverted. New line at end of file should be there.

Reply:
@meltinglava I'm not sure what's happened here as the lines appear to be the same :/ Why is it highlighting a change?

Reply:
Fixed.